app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    is_cp, command_to_read, command_to_write = command.split()
```

Comment:
what if command split does not return exactly three elements?

Reply:
but we have condition "Function take only one argument command, that is string with command cp, file name to copy and new file name, separated by spaces."